public/challenge-contributions/3/rpetrov.html

```diff
@@ -4,8 +4,211 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Challenge 3</title>
+
+  <style>
+    * {
+      border: 0;
+      padding: 0;
+      margin: 0;
+    }
+    html,
+    body {
+      min-height: 100vh;
+      background-color: #1d1d1d;
+      color: #fff;
+    }
+    .list {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(min(200px, 100%), 1fr));
+      gap: 16px;
+      padding: 32px;
+      max-width: 1200px;
+      margin: auto;
+    }
+    .scene {
+      display: flex;
+      place-content: center;
+      perspective: 1000px;
+    }
+    .card {
+      --radii: 8px;
+      
+      box-sizing: border-box;
+      position: relative;
+      width: 100%;
+      aspect-ratio: 3/4;
+      border-width: 2px;
+      border-style: solid;
+      border-color: rgb(255, 255, 255, 0.4);
+      background-color: #000;
+      transition: transform 1s ease, border-color 0.5s ease, box-shadow 0.5s ease;
+      transform-style: preserve-3d;
+      border-radius: var(--radii);
+      box-shadow: 0 0 8px rgba(255, 255, 255, .4);
+    }
+    .card:hover {
+      transform: rotateY(180deg);
+      box-shadow: 0 0 8px rgba(152, 2, 182, 0.6);
```

Comment:
Why the shadow is in the hover state? The shadow moves too?

Reply:
I am changing the color and the intensity of the shadow on hover. The shadow is on the card element (both sides)  and the element is rotating/moves